test/harness/assert-throws-native.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: >
+    Functions that throw instances of the specified native Error constructor
+    satisfy the assertion.
+---*/
+
+assert.throws(Error, function() {
+  throw new Error();
+});
+
+assert.throws(EvalError, function() {
+  throw new EvalError();
+});
+
+assert.throws(RangeError, function() {
+  throw new RangeError();
+});
+
+assert.throws(ReferenceError, function() {
+  throw new ReferenceError();
+});
+
+assert.throws(SyntaxError, function() {
+  throw new SyntaxError();
+});
+
+assert.throws(TypeError, function() {
+  throw new TypeError();
+});
+
+assert.throws(URIError, function() {
+  throw new URIError();
+});
```

Comment:
Agreed; this is already covered by https://github.com/bocoup/test262/blob/30ad8abdb5fea908c5f825f3b269752e873cb58d/test/harness/assert-throws-incorrect-ctor.js

```js
/*---
description: >
Functions that throw values whose constructor does not match the specified
constructor do not satisfy the assertion.
negative: Test262Error
---*/
assert.throws(Error, function() {
  throw new TypeError();
});
```